content/definition.md

```diff
@@ -5,10 +5,11 @@ template = "definition"
 
 Software is considered Ethical Source if it meets the following criteria:
 
-1. Freely distributed with source code and can be used or combined with other software without a royalty or fee.
-1. Developed in public view and welcoming of community contributions.
-1. The community of contributors is governed by a code of conduct that is consistently and fairly enforced.
-1. Creators have the right to prohibit use by individuals or organizations engaged in human rights violations or other behavior deemed unethical.
-1. Creators have the right to solicit reasonable and voluntary compensation from the communities or institutions that benefit from the software.
+1. It is distributed with source code and can be freely used or combined with other software.
+1. It is developed or maintained in public view and welcoming of public  contributions.
```

Comment:
Two spaces between "public" and "contributions!

Reply:
Do we need to be clearer on what "welcoming" means? Or are we happy for the moment to leave it somewhat vague, given the CoC requirement?

Reply:
Tha latter IMO